packages/contracts-bedrock/src/dispute/FaultDisputeGame.sol

```diff
@@ -389,7 +389,13 @@ contract FaultDisputeGame is IFaultDisputeGame, Clone, ISemver {
         // Uncontested claims are resolved implicitly unless they are the root claim. Pay out the bond to the claimant
         // and return early.
         if (challengeIndicesLen == 0 && _claimIndex != 0) {
-            _distributeBond(parent.claimant, parent);
+            // In the event that the parent claim is at the max depth, there will always be 0 subgames. If the
+            // `counteredBy` field is set and there are no subgames, this implies that the parent claim was successfully
+            // stepped against. In this case, we pay out the bond to the party that stepped against the parent claim.
+            // Otherwise, the parent claim is uncontested, and the bond is returned to the claimant.
+            address counteredBy = parent.counteredBy;
+            address recipient = counteredBy == address(0) ? parent.claimant : counteredBy;
+            _distributeBond(recipient, parent);
```

Comment:
Shouldn't the challenger who made the claim at max depth be the one to receive the bond as opposed to the actor who called step?

Reply:
Not necessarily - if the claim at max depth is countered, the person who disproved it with a step receives their bond. If it's uncountered, they get their bond back.

Reply:
Makes sense, sweet!